addons/sourcemod/scripting/mce/internal_functions.inc

```diff
@@ -59,15 +59,16 @@ NominateResult InternalNominateMap(char[] map, bool force, int owner)
 
 		RemoveFromArray(g_NominateList, 0);
 		RemoveFromArray(g_NominateOwners, 0);
-		g_sNominations[owner] = "";
+		ClearStoredNomination(owner_);
```

Comment:
I wonder if I should put this in the else condition. 🤔

Reply:
from what i understand it should stay here